Source/CvvInputView.swift

```diff
@@ -30,7 +30,7 @@ import UIKit
 
     private func setup() {
         textField.keyboardType = .numberPad
-        textField.textContentType = nil
+        textField.textContentType = .creditCardNumber
```

Comment:
@ioan-ghisoi-cko , Just one quick question do these bug fixes directly go inside master ? do we not have release branches ? or we use tags for different SDK releases ?